src/audio/kpb.c

```diff
@@ -1701,6 +1717,16 @@ static void kpb_init_draining(struct comp_dev *dev, struct kpb_client *cli)
 		/* Pause selector copy. */
 		comp_buffer_get_sink_component(kpb->sel_sink)->state = COMP_STATE_PAUSED;
 
+		if (!pm_runtime_is_active(PM_RUNTIME_DSP, PLATFORM_PRIMARY_CORE_ID))
+			pm_runtime_disable(PM_RUNTIME_DSP, PLATFORM_PRIMARY_CORE_ID);
+
+		comp_info(dev, "Scheduling draining task");
```

Comment:
can this be `comp_dbg()`? Doesn't seem to carry a lot of useful information beyond debugging

Reply:
This line is executed once before scheduling draining task. So it's just one line in the log. I think it is still useful to see if draining has started and finished.